test/date_time_parser_test.rb

```diff
@@ -53,4 +53,13 @@
     @date_parser.parse(date).must_equal date
     @time_parser.parse(time).must_equal time
   end
+
+  it "raises ArgumentError when parsing a bad string" do
+    proc { @date_parser.parse("asdf") }.must_raise ArgumentError
```

Comment:
Please make sure that it's the right `ArgumentError` by also checking for the exception message. Same goes for the two assertions in the following block.